src/utils.js

```diff
@@ -101,17 +101,14 @@ function Protocol (log) {
 }
 
 function getIdAndValidate (pub, id, cb) {
```

Comment:
Why not doing async await here as well?

Reply:
It didn't appear to be part of the public API and is only invoked in the context of a `socket.io-pull-stream` handler which is all callbacks.